TombEngine/Scripting/Internal/TEN/Objects/ObjectsHandler.h

```diff
@@ -64,6 +64,45 @@ class ObjectsHandler : public ScriptInterfaceObjectsHandler
 			return std::make_unique<R>(std::get<R::IdentifierType>(m_nameMap.at(name)));
 	}
 
+	template <typename R, char const* S>
+	std::vector <std::unique_ptr<R>> GetMoveablesBySlot(GAME_OBJECT_ID objID)
+	{
+		std::vector<std::unique_ptr<R>> items = {};
+		for (auto& [key, val] : m_nameMap)
+		{
+			if (!std::holds_alternative<std::reference_wrapper<MESH_INFO>>(val))
+			{
+				auto* item = &g_Level.Items[GetIndexByName(key)];
+				GAME_OBJECT_ID objectNumber = item->ObjectNumber;
+
+				if (objID == objectNumber)
+					items.push_back(GetByName<Moveable, ScriptReserved_Moveable>(key));
+			}
+		}
+
+		return items;
+	}
+
+	template <typename R, char const* S>
+	std::vector <std::unique_ptr<R>> GetStaticsById(int slot)
+	{
+		std::vector<std::unique_ptr<R>> items = {};
+		for (auto& [key, val] : m_nameMap)
+		{
+			if (std::holds_alternative<std::reference_wrapper<MESH_INFO>>(val))
+			{
+				auto meshInfo = std::get<std::reference_wrapper<MESH_INFO>>(val).get();
+				int staticNumber = meshInfo.staticNumber;
+				int slotNr = slot;
+
+				if (staticNumber == slotNr)
+					items.push_back(GetByName<Static, ScriptReserved_Moveable>(key));
```

Comment:
It probably doesn't make a functional difference here but as we are getting a static it would be more correct to use `ScriptReserved_Static` instead of `ScriptReserved_Moveable`